addons/settings/XEH_preInit.sqf

```diff
@@ -37,7 +37,7 @@ if (isServer) then {
         } forEach _settings;
 
         // report specific medical settings
-        private _log = format ["[AAA=%1] [aDmgPass=%2]", AAA_VAR_MOD_ENABLED, ace_medical_engine_damagePassThroughEffect toFixed 2];
+        private _log = format ["[aDmgPass=%1]", ace_medical_engine_damagePassThroughEffect toFixed 2];
```

Comment:
Already making a change here as part of #550. If you want it this way, let me know and I'll remove it from that PR to avoid the merge conflict.